### PR TITLE
Add Log component

### PR DESCRIPTION
**Pull request by Ikilonchic**

### CODE REVIEW

**Comment by dkupchyk** [comment]
Is it possible to make the background of the page when log apperars a bit darker in order to highlight the log?

File: src/scripts/components/LogMessage.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  hide() {
+    this.rootElement.classList.remove('log-message_show');
+  }
+
+  render() {
+    this.rootElement.innerHTML = this.buildView();
+  }
+
+  buildView() {
+    return `<div class="log-message__text-wrap log-message_type-${messageTypes.includes(this.type) ? this.type : 'default'}">
```
Comment (dkupchyk) [inline src/scripts/components/LogMessage.js]: Suggest moving ${messageTypes.includes(this.type) ? this.type : 'default'} into a variable and then using it in the return statement.

Comment (Ikilonchic) [reply src/scripts/components/LogMessage.js]: Yes, I didn't see this.

File: src/scripts/components/LogMessage.js [hunk 2]
```diff
@@ -12,41 +12,62 @@ class LogMessage {
...
   }
 
   hide() {
     this.rootElement.classList.remove('log-message_show');
   }
 
+  addMessage(message, type = 'error', time = 2000) {
+    const id = (this.messageList[this.messageList.length - 1]?.id + 1) || 0;
+
+    this.messageList = this.messageList.slice(-4);
```
Comment (dkupchyk) [inline src/scripts/components/LogMessage.js:41]: Why exactly slice with -4? Is it for the ability to see only the messages that fit the screen?

Comment (Ikilonchic) [reply src/scripts/components/LogMessage.js:41]: I select 5 most relevant posts.

**Pull request merged**